src/app/pages/landpage/landpage.component.html

```diff
@@ -14,10 +14,10 @@ <h1>Study Site</h1>
     <h3>Sign in</h3>
     <div class="login">
       <p>Email</p>
-      <input type="text" placeholder="Email">
+      <input type="email" class="lInput" [(ngModel)]="user.email"  placeholder="Email" name="email">
```

Comment:
Class names should be lower case with dashes "-" betwean words. Example "lInput" --> "l-input"